features/step_definitions/basic_steps.rb

```diff
@@ -1 +1,17 @@
-# put step definitions in this file
\ No newline at end of file
+When(/^I visit the site$/) do
+  visit '/'
+end
+
+
+Then(/^I want to see "([^"]*)"$/) do |text|
+  expect(page).to have_content text
+end
+
+When(/^I visit the name page for "([^"]*)"$/) do |name|
+  visit '/name/Amber'
+end
+
+Then(/^I should create a new user with name "([^"]*)"$/) do |name|
+  User.new 'Amber'
+  expect(user.name).to eq name
```

Comment:
Same here. `name` should be used when creating a User. Also, you need to use the attribute name and actually save the user in the database.

Reply:
This step needs to be rewritten. Where do you create the variable `user`?